src/sources/kubernetes_logs/mod.rs

```diff
@@ -525,6 +525,7 @@ impl Source {
                     Err(error) => emit!(KubernetesLifecycleError {
```

Comment:
I'm just assuming here that this is a non-recoverable failure that results in dropped events. I don't actually know for sure.